__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require("supertest");
+const db = require("../db/data/test-data");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+const app = require("../app");
+const connection = require("../db/connection.js");
+
+beforeAll(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return connection.end();
+});
+
+describe("api/topics", () => {
+  describe("GET - api/topics", () => {
+    test("200-endpoint responds with 200 status and an array of topic objects", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const topics = response.body.topics;
+          topics.forEach((topic) => {
```

Comment:
needs a length check otherwise you could end up with a test passing when receiving no topics (forEach won't run)